lib/Raven/Serializer.php

```diff
@@ -67,17 +67,44 @@ public function __construct($mb_detect_order = null)
      * sanitization and encoding.
      *
      * @param mixed $value
+     * @param array $context
+     * @return string|bool|double|int|null|object|array
```

Comment:
If I have many available types but these are not **all** valid options I chouse to list them

https://github.com/nokitakaze/ascetcms/blob/49049665babfd32564cc567d69ea4b41e9457d3e/functions/smodel.php#L1326
https://github.com/nokitakaze/ascetcms/blob/49049665babfd32564cc567d69ea4b41e9457d3e/functions/sdbcriteria.php#L10